e2e/cypress/tests/plugins/external_request.ts

```diff
@@ -20,7 +33,7 @@ module.exports = async ({baseUrl, user, method = 'get', path, data = {}}) => {
         });
 
         const setCookie = response.headers['set-cookie'];
-        setCookie.forEach((cookie) => {
+        (setCookie as any).forEach((cookie: string) => {
```

Comment:
shouldn't this be `const setCookie: string[] = response.headers['set-cookie'];` and then you can write `setCookie.forEach((cookie: string) => {`